openwisp_radius/api/views.py

```diff
@@ -326,6 +353,9 @@ def get_queryset(self):
 
 
 class BatchView(TokenAuthorizationMixin, generics.CreateAPIView):
+    # TODO:N Who can use obtaintokenview?
+    # Should they be allowed to use batchview to create
+    # users? (Uses TokenAuthentication)
```

Comment:
`BatchView` uses `TokenAuthentication`, so basically anyone having access to `ObtainAuthTokenView` can have access to `BatchView`; this looks undesirable because my understanding is that `ObtainAuthTokenView` can be accessed by user.

I want to confirm the same before making any change.

Reply:
leave this untouched, there's a dedicated issue for this view: https://github.com/openwisp/openwisp-radius/issues/94